rmgpy/data/kinetics/groups.py

```diff
@@ -196,6 +196,8 @@ def estimateKineticsUsingGroupAdditivity(self, template, referenceKinetics, dege
             kinetics.kdata.value_si *= degeneracy
         elif isinstance(kinetics, Arrhenius):
             kinetics.A.value_si *= degeneracy
+        elif isinstance(kinetics, ArrheniusEP):
+            kinetics.A.value_si *= degeneracy
```

Comment:
The fix looks good. Could you squash this commit with the updated line of code `switch degeneracy adjustments to use changeRate`? Having fewer modifications in the same PR  will make traceback easier. When rewording up the commit message, if you think having the previous squashed commit names and descriptions is not useful, you may consider cleaning up.